tests/fideslang/test_validation.py

```diff
@@ -717,43 +717,37 @@ def test_return_all_elements_on_array_field(self):
         )
 
     def test_data_categories_at_object_level(self):
-        with pytest.raises(ValidationError) as exc:
-            DatasetField(
-                name="test_field",
-                data_categories=["user"],
-                fides_meta=FidesMeta(
-                    references=None,
-                    identify=None,
-                    primary_key=False,
-                    data_type="object",
-                    length=None,
-                    return_all_elements=None,
-                    read_only=None,
-                ),
-                fields=[DatasetField(name="nested_field")],
-            )
-        assert_error_message_includes(
-            exc, "Object field 'test_field' cannot have specified data_categories"
+        # Data categories at the object level ARE allowed now
+        DatasetField(
```

Comment:
should we assign this to a variable and assert that the field both has a category and a subfield? something like `assert field.data_categories == ["user"]` and `assert field.fields == ...`

Reply:
yes good call, added in latest commit